### PR TITLE
GH-1250 - DERM_OBJ_WATCH_FIX

### PR DESCRIPTION
**Pull request by mlzummo**
Fixes object watch for firefox
Fixes GH-1250

Signed-off-by: Mathew Zummo <mzummo@gmail.com>

### CODE REVIEW

**Comment by mlzummo** [comment]
@GordonSmith @dtsnell4 please review

File: src/layout/Grid.js [hunk 1]
```diff
@@ -314,7 +314,7 @@
                 d
                    .target(this)
                 ;
-                d.__grid_watch = d.watch(function (key, newVal, oldVal) {
+                d.__grid_watch = d.watch(null,function (key, newVal, oldVal) {
```
Comment (GordonSmith) [inline src/layout/Grid.js]: watch only takes one param.

**Comment by mlzummo** [comment]
@GordonSmith Please Re-Review ... made suggested changes ..

File: src/common/Widget.js [hunk 2]
```diff
@@ -359,7 +359,7 @@
         };
     };
 
-    Widget.prototype.watch = function (func) {
+    Widget.prototype.monitor = function (func) {
```
Comment (GordonSmith) [inline src/common/Widget.js]: This change will need to be propagated throughout all the sources...

Comment (mlzummo) [inline src/common/Widget.js]: i believe i did already

Comment (mlzummo) [inline src/common/Widget.js]: your right i messed 1 or 2

**Comment by dtsnell4** [comment]
OK w me

**Pull request merged**